deepmd/utils/econf_embd.py

```diff
@@ -151,11 +166,13 @@
     (6, "p"),
     (6, "d"),
     (7, "s"),
+    (7, "p"),
 ]
 
 maxn = 7
 maxl = maxn
 maxm = 2 * maxl + 1
+ECONF_DIM = 59
```

Comment:
should be get from the dims of `electronic_configuration_embedding` rather than providing a constant.